noir-projects/noir-contracts/contracts/token_contract/src/main.nr

```diff
@@ -410,29 +407,27 @@ pub contract Token {
         let token = Token::at(context.this_address());
 
         // We prepare the private balance increase (the partial note).
-        let hiding_point_slot = _prepare_private_balance_increase(from, to, &mut context, storage);
+        let partial_note = _prepare_private_balance_increase(from, to, &mut context, storage);
```

Comment:
It's technically a partial_note_commitment, given that the partial_note is actually a struct of information. (Sorry - I was referring to this PR for another reason, then saw this).

Reply:
Yeah this is ultimately an issue due to us not neatly following Rust's newtype idiom. `PartialUintNote` is a struct with a single value.